kas/patches/poky_image_bbclass_rootfs_size_messages.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/meta/classes/image.bbclass b/meta/classes/image.bbclass
```

Comment:
this patch is not mandatory as it obviously doesn't change functionality, it only prints image sizes during build which would be output only if debugging is turned on which would result in myriads of messages where these messages would be the needle in the haystack ...
I can throw it out if you don't see any benefit in it, otherwise I could create a combined patch.

Reply:
@JanZachmann

Reply:
combined patch created